sysbuild/Kconfig.lwm2m_carrier

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+menuconfig LWM2M_CARRIER_DIVIDED_DFU
+	bool "Proprietary LwM2M Carrier divided DFU file generation"
+	depends on BOOTLOADER_MCUBOOT
+	help
+	  Generate proprietary application update files required for the LwM2M Carrier divided
+	  FOTA procedure.
+
+if LWM2M_CARRIER_DIVIDED_DFU
+
+config LWM2M_CARRIER_DIVIDED_DFU_MAX_FILE_SIZE
+	int "Divided DFU maximum file size"
+	range 1 100
```

Comment:
1...256?

Reply:
Was SoftBank ok with that for application FOTA?

Reply:
Yes, aligned with MFW delta FOTA limit, but please keep using 100KB as default